posthog/management/commands/warm_team_metadata_cache.py

```diff
@@ -0,0 +1,104 @@
+"""
+Management command to warm the team metadata cache.
+
+Usage:
+    # Initial cache warm (preserves existing caches)
+    python manage.py warm_team_metadata_cache
+
+    # Schema changes (invalidates all caches first)
+    python manage.py warm_team_metadata_cache --invalidate-first
+
+    # Custom batch size and TTL range
+    python manage.py warm_team_metadata_cache --batch-size 200 --min-ttl-days 6 --max-ttl-days 8
+"""
+
+from django.core.management.base import BaseCommand
+
+from posthog.storage.team_metadata_cache import warm_all_team_caches
+
+
+class Command(BaseCommand):
+    help = "Warm team metadata cache for all teams (initial build or schema migration)"
+
```

Comment:
I think we should add an argument to warm a specific team. For example, suppose we discover that for some reason, team `12345`'s cache entry is out of date or not present. Then we could run this command passing in `--team-id 12345` and just warm that team.